### PR TITLE
Add toolbar elevation

### LINKED ISSUE #347: [Assigned] Add toolbar elevation in sponsors page and contributors page
## Overview (Required)

20180120_182845_screenshot_2018-01-20-18-27-12 jpg 1080x1920

Contributors page as well.

### PR DESCRIPTION
**Pull request by tebasakyu**
## Issue
- close #347

## Overview (Required)
- Added drop shadow. This can work below lollipop.

## Links
- [Toolbar-Elevation-Pre-Lollipop](https://github.com/vipulasri/Toolbar-Elevation-Pre-Lollipop)

### CODE REVIEW

**Comment by tebasakyu** [comment]
I saw other activities has same issue.  
And fragments on MainActivity are set `android:elevation`, but this can work lollipop or more.  

Can I fix above in this issue?

File: app/src/main/res/layout/activity_contributor.xml [hunk 1]
```diff
@@ -38,6 +38,15 @@
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/toolbar"
                 />
+
+            <View
```
Comment (konifar) [inline app/src/main/res/layout/activity_contributor.xml]: 🙋‍♂️ 
How about creating another layout and use `<include layout="" />` tag?

**Comment by takahirom** [comment]
Thanks 👍 
I think KitKat users are decreasing. So I think that it is okay to ignore the detailed design. 🤔 
It is to make shadow animation instead of abandoning compatibility 👍

**Comment by konifar** [comment]
I thought it's enough to add shadow only for lollipop.
Actually `minSdk` of the product in the company I belongs to is 21 😉

**Comment by takahirom** [comment]
Sorry 🙇 
Please fix conflict

**Comment by tebasakyu** [comment]
I added elevation on style.xml.
Please check again 🙇

**Comment by konifar** [comment]
LGTM! 👍  Thanks!

**Pull request merged**